bin/cmd.js

```diff
@@ -5,8 +5,12 @@ var match = process.version.match(/v(\d+)\.(\d+)/)
 var major = parseInt(match[1], 10)
 var minor = parseInt(match[2], 10)
 
-if (major >= 11 || (major === 10 && minor >= 12)) {
-  require('standard-engine').cli(require('../options'))
+if (major >= 12 || (major === 12 && minor >= 20)) {
+  import('standard-engine').then(mod => {
```

Comment:
This might be small, but to improve clarity, I would not call this `mod` but `module` or `standardEngine` instead.

What do you think ? :smile:

Reply:
sure, can change it. 
just started naming things mod just b/c Deno [did it](https://deno.land/manual/contributing/style_guide#do-not-use-the-filename-codeindextscodecodeindexjscode)... short for module